src/scripting_api/doc.js

```diff
@@ -272,7 +274,7 @@ class Doc extends PDFObject {
   }
 
   get external() {
-    return true;
+    return DOC_EXTERNAL;
```

Comment:
Maybe you could just add a small explanation on why we aren't "correctly" implementing this getter.

Reply:
Sure, fixed now!